data/cleansed/countries.json

```diff
@@ -1546,6 +1532,13 @@
     "iso_3166_3": "THA",
     "currency": "THB"
   },
+  {
+    "name": "The former Yugoslav Republic of Macedonia",
```

Comment:
This looks like a revert of https://github.com/flowcommerce/json-reference/commit/c0c144a7da24be65a833779e765cd01a7be1283c, we should undo this change